pom.xml

```diff
@@ -343,9 +343,6 @@
           <warName>${project.artifactId}</warName>
         </configuration>
       </plugin>
-
```

Comment:
I meant that tags on the second level should have one line between them, tags on every other level should not. So if you coud add one empty line between 345-346 that would be great